src/Console/Command/SelfUpdate.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Povils\PHPMND\Console\Command;
+
+use Povils\PHPMND\Console\Application;
+use Symfony\Component\Console\Command\Command as BaseCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Humbug\SelfUpdate\Updater;
+use Humbug\SelfUpdate\Strategy\GithubStrategy;
+
+class SelfUpdate extends BaseCommand
+{
+
+    /**
+     * Packagist package name
+     */
+    const PACKAGE_NAME = 'povils/phpmnd';
+
+    /**
+     * This is the remote file name, not local name.
+     */
+    const FILE_NAME = 'phpmnd.phar';
+
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * @var string
+     */
+    protected $version;
+
+    /**
+     * Setup command and arguments.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setDescription('Update phpmnd.phar to most recent stable build.')
+            ->addOption(
+                'stable',
+                's',
+                InputOption::VALUE_NONE,
```

Comment:
This can probably be deleted. If support is needed for non-stable releases (alphas, betas, RCs) alternative stability flags can be passed into phar-updater.